04-Intro_to_Vue/assignment/vue-dub3/src/components/vue-Pics.vue

```diff
@@ -0,0 +1,29 @@
+<script setup>
+import { ref } from 'vue'
+
+const name = ref('Pics')
+
+const images = ref(["Dog Image", "Painting Image"])
+</script>
+
+<template>
+  <h1>PICS</h1>
+  <ul>
+    <li v-for="image in images" :key="image">
+      {{ image }}
+    </li>
+<!-- Need to add these images to the above array -->
```

Comment:
Yeah. You're close here. Just need to think how you can leverage the <img /> tag and pass in the src based on the array's information